### DISCUSSION TITLE
New Features Release

### DISCUSSION THREAD

**Discussion by Sid72020123** [Sid72020123/scratchconnect#45] [Polls]
I have made some new features like making this library work on Replit using proxy and Cloud Requests feature (still in development). I want to know when should I release those features?
I have much more cool features to make. But the development is taking time.
Vote for options below so that I will find the date when to release them...

**Comment by Sid72020123** [comment]
I will see the results of this poll till 10th September...

**Comment by Sid72020123** [comment]
Ok. I see that the 2nd option has the most votes so I will release the newer version after making more new features. 
But it may take time as I am busy(due to school)

**Reply by Quantum-Codes** [reply]
Great
